src/table.jl

```diff
@@ -145,4 +145,5 @@ Get a vector of the geometries in a shapefile `Table`, without any metadata.
 """
 shapes(t::Table) = shapes(getshp(t))
 
+GeoInterface.extent(t::Table) = GeoInterface.extent(getshp(t))
```

Comment:
Here we could also implement `isfeature`, `geometry` and `properties` right?

Reply:
Well a table is a `FeatureCollection`, so they would go above on `Row`

Reply:
I've added them for `Row`